### PR TITLE
New post: Server softwares

### PR DESCRIPTION
**Pull request by Mocab**
will be updated soon to include all other server softwares (bedrock, teamspeak, terraria, etc)

### CODE REVIEW

File: _posts/minecraft/general/2021-8-22-server-softwares.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title:  "Server softwares"
+categories: Minecraft
+tag: mcgeneral
+permalink: /minecraft/general/server-software/
```
Comment (itsjuoum) [inline _posts/minecraft/general/2021-8-22-server-softwares.md:6]: This is an article that talks about multiple server softwares. I think that the permalink should be `/minecraft/general/server-softwares/`

File: _posts/minecraft/general/2021-8-22-server-softwares.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title:  "Server softwares"
```
Comment (itsjuoum) [inline _posts/minecraft/general/2021-8-22-server-softwares.md:3]: Capitalize the s in "softwares"

File: _posts/minecraft/general/2021-8-22-server-softwares.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+categories: Minecraft
+tag: mcgeneral
+permalink: /minecraft/general/server-software/
+---
+
+# Server softwares
```
Comment (itsjuoum) [inline _posts/minecraft/general/2021-8-22-server-softwares.md:9]: Capitalize the s in "softwares"

File: _posts/minecraft/general/2021-8-22-server-softwares.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
```
Comment (itsjuoum) [inline _posts/minecraft/general/2021-8-22-server-softwares.md:10]: I believe that adding a quick description to this article is important. I came up with:

Choosing your server's software is a very important task. This article will help you choose the software that fits your server's needs.

File: _posts/minecraft/general/2021-8-22-server-softwares.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+## 1. PaperMC
```
Comment (itsjuoum) [inline _posts/minecraft/general/2021-8-22-server-softwares.md:11]: There's no need for an ordered list in this article. I think that removing the numeration will help make it look cleaner as well.

**Review by itsjuoum — CHANGES_REQUESTED** [review]

**Comment by itsjuoum** [comment]
I've also noticed that the article name format seems incorrect. It's missing a zero behind the 8 (month). It should look like this: `2021-08-22-server-softwares.md`

File: _posts/minecraft/general/2021-8-22-server-softwares.md (hunk 1 above)
Comment (Mocab) [reply _posts/minecraft/general/2021-8-22-server-softwares.md:6]: We should leave it Minecraft, we can move it once other server softwares are added

File: _posts/minecraft/general/2021-8-22-server-softwares.md (hunk 5 above)
Comment (Mocab) [reply _posts/minecraft/general/2021-8-22-server-softwares.md:11]: The numbers in the list correspond to the number in the server setup script

Comment (itsjuoum) [reply _posts/minecraft/general/2021-8-22-server-softwares.md:11]: Ah, understood.

File: _posts/minecraft/general/2021-8-22-server-softwares.md (hunk 1 above)
Comment (itsjuoum) [reply _posts/minecraft/general/2021-8-22-server-softwares.md:6]: I didn't move it from Minecraft, I simply added an s to make it plural.

Comment (Mocab) [reply _posts/minecraft/general/2021-8-22-server-softwares.md:6]: Oh my bad

File: _posts/minecraft/general/2021-8-22-server-softwares.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+Paper MC is a fork of bukkit and spigot that is optimized and allows you to use plugins, both spigot and bukkit plugins work, It also fixes some Vanilla bugs and exploits!
```
Comment (DaddyChimken) [inline _posts/minecraft/general/2021-8-22-server-softwares.md:12]:
```suggestion
[PaperMC](https://papermc.io) is a fork of Spigot that is optimized and allows you to use Spigot and Bukkit plugins. It also fixes some Vanilla bugs and exploits!
```

File: _posts/minecraft/general/2021-8-22-server-softwares.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+## 2. Purpur
+Purpur is a Fork of Paper and Tuinity that is very optimized, it has many extra features and supports spigot and bukkit plugins. We recommend using this server software.
```
Comment (DaddyChimken) [inline _posts/minecraft/general/2021-8-22-server-softwares.md:15]:
```suggestion
[Purpur](https://purpur.pl3x.net/) is a Fork of Tuinity that is very optimized. It contains extra features and supports Spigot and Bukkit plugins. We recommend using this server software.
```

File: _posts/minecraft/general/2021-8-22-server-softwares.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+
+## 3. Fabric
+Fabric is a Minecraft software that supports fabric mods and is lightweight.
```
Comment (DaddyChimken) [inline _posts/minecraft/general/2021-8-22-server-softwares.md:18]:
```suggestion
[Fabric](https://fabricmc.net/) is a lightweight Minecraft software used for Fabric mods.
```

File: _posts/minecraft/general/2021-8-22-server-softwares.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+
+## 4. Vanilla
+Vanilla is the default Minecraft software, it does not support plugins or mods.
```
Comment (DaddyChimken) [inline _posts/minecraft/general/2021-8-22-server-softwares.md:21]:
```suggestion
Vanilla is the default software provided by Minecraft. It does not support plugins or mods.
```

File: _posts/minecraft/general/2021-8-22-server-softwares.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+
+## 5. Vanilla snapshot
+Vanilla snapshot is the default Minecraft server, it does not support plugins or mods, use this if you want your server's version to be a snapshot.
```
Comment (DaddyChimken) [inline _posts/minecraft/general/2021-8-22-server-softwares.md:24]:
```suggestion
## 5. Vanilla Snapshot
Vanilla Snapshot is a testing version of Minecraft. Snapshot's usually contain bugs and are not recommended for production use. Just like Vanilla, they do not support plugins or mods.
```

File: _posts/minecraft/general/2021-8-22-server-softwares.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+
+## 6. Tuinity
+Tuinity is a fork of Paper that is optimized, it supports spigot and bukkit plugins.
```
Comment (DaddyChimken) [inline _posts/minecraft/general/2021-8-22-server-softwares.md:27]: Please remove Tuinity from this list.

All of Tuinity's projects and patches will be merged into Paper.

View the image below from PaperMC's announcement channel:
image

File: _posts/minecraft/general/2021-8-22-server-softwares.md [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+
+## 7. Forge
+Forge is a Minecraft software that supports forge mods.
```
Comment (DaddyChimken) [inline _posts/minecraft/general/2021-8-22-server-softwares.md:30]:
```suggestion
[Forge](https://files.minecraftforge.net/net/minecraftforge/forge) is a Minecraft software that allows players to use Forge mods.
```

File: _posts/minecraft/general/2021-8-22-server-softwares.md [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+
+## 8. Cuberite 
+Cuberite is a Minecraft server software that is optimized and supports plugins, it does **not** support spigot and bukkit plugins.
```
Comment (DaddyChimken) [inline _posts/minecraft/general/2021-8-22-server-softwares.md:33]:
```suggestion
[Cuberite](https://cuberite.org/) is an optimized Minecraft server software used for Cuberite plugins.
```

**Review by DaddyChimken — CHANGES_REQUESTED** [review]

File: _posts/minecraft/general/2021-08-22-server-softwares.md [hunk 14]
```diff
@@ -0,0 +1,31 @@
...
+categories: Minecraft
+tag: mcgeneral
+permalink: /minecraft/general/server-softwares/
+---
+
+# Server Softwares
+Choosing your server's software is a very important task. This article will help you choose the software that fits your server's needs.
+
+## PaperMC
+[PaperMC](https://papermc.io) is a fork of Spigot that is optimized and allows you to use Spigot and Bukkit plugins. It also fixes some Vanilla bugs and exploits!
```
Comment (DaddyChimken) [inline _posts/minecraft/general/2021-08-22-server-softwares.md:13]: No need to mention what plugins you can run imo.
If you already mention Paper is a fork of Spigot, then it makes sense you can run Spigot/Bukkit plugins.

```suggestion
[PaperMC](https://papermc.io) is an optimized fork of Spigot that fixes some Vanilla bugs and exploits!
```

File: _posts/minecraft/general/2021-08-22-server-softwares.md [hunk 15]
```diff
@@ -0,0 +1,31 @@
...
+
+## Purpur
+[Purpur](https://purpur.pl3x.net/) is a Fork of Tuinity that is very optimized. It contains extra features and supports Spigot and Bukkit plugins. We recommend using this server software.
```
Comment (DaddyChimken) [inline _posts/minecraft/general/2021-08-22-server-softwares.md:16]:
```suggestion
[Purpur](https://purpur.pl3x.net/) is an optimized Fork of Tuinity. It contains extra features and supports Spigot and Bukkit plugins. We recommend using this.
```

File: _posts/minecraft/general/2021-08-22-server-softwares.md [hunk 16]
```diff
@@ -0,0 +1,31 @@
...
+
+## Fabric
+[Fabric](https://fabricmc.net/) is a lightweight Minecraft software used for Fabric mods.
```
Comment (DaddyChimken) [inline _posts/minecraft/general/2021-08-22-server-softwares.md:19]:
```suggestion
## FabricMC
[FabricMC](https://fabricmc.net) is a lightweight Minecraft software used for Fabric mods.
```

**Review by DaddyChimken — CHANGES_REQUESTED** [review]

**Review by DaddyChimken — APPROVED** [review]

**Review by itsjuoum — APPROVED** [review]

**Pull request merged**